new_text_string.sh

```diff
@@ -38,17 +38,22 @@ if [ -z "${MYSQLD}" ]; then
 fi
 
 # The */ in the */*core* core search pattern is for to the /node1/ dir setup for cluster runs
```

Comment:
Please move the comment line 40 to inside the `else` block also

Reply:
i.e. just above the LATEST_CORE line

Reply:
Discussed; the line is superfluous and incorrect, and can be removed alltogether.